### PR TITLE
dev

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
# Description

This PR aims at adding this feature...

# Proposed Changes

I changed the `foo` function so that ...

### CODE REVIEW

File: R/report.data.frame.R [hunk 1]
```diff
@@ -475,10 +480,14 @@ report.numeric <- function(model, median = FALSE, centrality = TRUE, dispersion
   }
 
   # Missing
-  if (missing_percentage == TRUE) {
-    table <- dplyr::select(table, -one_of("n_Missing"))
-  } else {
-    table <- dplyr::select(table, -one_of("percentage_Missing"))
+  if(!is.null(missing_percentage)){
+    if (missing_percentage == TRUE) {
+      table <- dplyr::select(table, -one_of("n_Missing"))
```
Comment (strengejacke) [inline R/report.data.frame.R]: We have the small helper function `.remove_columns()` to avoid dplyr, already used elsewhere in _report_.

File: R/report_regressions.R [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-  if (length(c(ci)) > 1) {
+  if(length(c(ci)) > 1){
     warning(paste0("report does not support multiple `ci` values yet. Using ci = ", ci[1]), ".")
     ci <- ci[1]
   }
 
   # Default effsize
-  if (!is.null(effsize) && effsize == "default") {
-    if (info$is_binomial) {
+  if(!is.null(effsize) && effsize == "default"){
```
Comment (strengejacke) [inline R/report_regressions.R]: Why did you remove the white paces before `{`?

Comment (DominiqueMakowski) [reply R/report_regressions.R]: did not voluntarily, maybe I overwrote by mistake a non-stylered version
